test/snapshots/app.test.ts.snap

```diff
@@ -201,6 +201,8 @@ exports[`app getFrontendApp please review routes and update snapshot when adding
 	query,expressInit,elapsedTimeMetrics,sentryRequestMiddleware,urlencodedParser,jsonParser,cookieParser,LogMiddleware,_cookieSession,jiraSymmetricJwtMiddleware,JiraSecurityWorkspacesPost
 :GET ^/?(?=/|$)^/jira/?(?=/|$)^/?$
 	query,expressInit,elapsedTimeMetrics,sentryRequestMiddleware,urlencodedParser,jsonParser,cookieParser,LogMiddleware,_cookieSession,csrf,jiraSymmetricJwtMiddleware,jiraAdminPermissionsMiddleware,JiraGet
+:GET ^/?(?=/|$)^/jira/?(?=/|$)^/subscriptions/backfill-status/?$
+	query,expressInit,elapsedTimeMetrics,sentryRequestMiddleware,urlencodedParser,jsonParser,cookieParser,LogMiddleware,_cookieSession,JiraGetConnectionsBackfilStatus
```

Comment:
I believe this must be under jiraSymmetricJwtMiddleware, otherwise the endpoint is not protected (public) and anyone can query it

Reply:
Yes it should be! 
added 2 middlewares as suggested by Gary:
jiraSymmetricJwtMiddleware, jiraAdminPermissionsMiddleware,